app.js

```diff
@@ -2,3 +2,58 @@
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
 //  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
+
+var express = require('express');
+var app = express();
+var foodTrucks = require('./trucks');
+
+app.use(express.static('public'));
+
+app.get('/', function (request, response) {
```

Comment:
This is unnecessary. Using express.static as middle-ware processes the root route so this code won't do anything.